assets/components/atoms/social-feed-item/social-feed-item.twig

```diff
@@ -0,0 +1,16 @@
+<div class="social-feed-item">
+
+  <span class="social-icon social-icon-twitter social-icon-round">
+    {% include '../../atoms/icon/icon.twig' with { icon: 'icon-twitter' } %}
+  </span>
+
+  <p class="social-feed-header"><strong>@epflENAC</strong> <span class="text-small">il y a 8 heures</span></p>
+  <p>A Berlin, les étudiantes et étudiants de Dominique Perrault exposent leurs travaux à la galerie Aedes Architecture Forum du 27 janvier au 8 mars. @dpa_official http://www.aedes..</p>
+
+  {% include '../../atoms/picture/picture.twig' %}
```

Comment:
It's the wrong image size, you can create a third desktop picture using the [calculator](http://yago.io/picture-tag-generator/)